core-header-panel.css

```diff
@@ -31,6 +31,10 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
   -webkit-overflow-scrolling: touch;
 }
 
+#mainContainer {
+  transform: translateZ(0);
```

Comment:
This should also have a `-webkit-transform` version before it. Safari, Mobile Safari, and Android Browser still use the prefix: http://caniuse.com/#search=transform